nuxt.config.js

```diff
@@ -227,16 +238,10 @@ export default {
       'LayoutPlugin',
       'LinkPlugin',
       'ListGroupPlugin',
-      'MediaPlugin',
       'ModalPlugin',
       'NavbarPlugin',
-      'NavPlugin',
-      'PaginationNavPlugin',
       'SidebarPlugin',
```

Comment:
The SidebarPlugin is only used in the PageHeader component so we could also only import it there.

Reply:
We could, but it's rendered on every page so it doesn't make a difference to the overall payload.